Source/WebKit/UIProcess/ios/forms/WKFormSelectControl.h

```diff
@@ -43,12 +43,16 @@ CGFloat adjustedFontSize(CGFloat textWidth, UIFont *, CGFloat initialFontSize, c
 - (void)selectRow:(NSInteger)rowIndex inComponent:(NSInteger)componentIndex extendingSelection:(BOOL)extendingSelection;
 - (BOOL)selectFormAccessoryHasCheckedItemAtRow:(long)rowIndex;
 @property (nonatomic, readonly) NSString *selectFormPopoverTitle;
+
+- (NSArray<NSString *> *)menuItemTitles;
```

Comment:
Nit - I think we have a slight preference for:

```
@property (nonatomic, readonly) NSArray<NSString *> *menuItemTitles;
```

Reply:
Done. I've kept the protocol method the same though, since I think it's less common to use properties there (though open to changing if desired).